sync-core/src/test/java/com/cloudant/sync/datastore/AttachmentTest.java

```diff
@@ -66,6 +62,11 @@ public void setAndGetAttachmentsTest() throws Exception {
             SavedAttachment savedAtt2 = (SavedAttachment) datastore.attachmentsForRevision
                     (newRevision).get(0);
             Assert.assertArrayEquals(expectedSha1, savedAtt2.key);
+
+            //Assert file length and saved attachment length
+            long expectedLength = f.length();
+            Assert.assertEquals(expectedLength,savedAtt.getSize());
+            Assert.assertEquals(expectedLength,savedAtt2.getSize());
```

Comment:
This is to confirm that the original file (variable f) is the same length as the saved attachment file length.  In the addAttachment method in AttachmentManager, the length from total bytes in PreparedAttachment is put into the ContentValue variable.  SavedAttachment retrieves this length when you call the getSize method. 
This test can also be useful when we add encryption to attachments and need to check that the decrypted length is the same as the original file length.